docs/excel/excel-add-ins-pivottables.md

```diff
@@ -221,7 +222,142 @@ Excel.run(function (context) {
 });
 ```
 
-## Slicers
+## Filter a PivotTable
+
+The primary method for filtering PivotTable data is with PivotFilters. Slicers offer an alternate, less flexible filtering method. 
+
+[PivotFilters](/javascript/api/excel/excel.pivotfilters) filter data based on a PivotTable's four [hierarchy categories](#hierarchies) (filters, columns, rows, and values). There are four types of PivotFilters, allowing calendar date-based filtering, string parsing, number comparison, and filtering based on a custom input. 
+
+[Slicers](/javascript/api/excel/excel.slicer) can be applied to both PivotTables and regular Excel tables. When applied to a PivotTable, slicers function like a [PivotManualFilter](#pivotmanualfilter) and allow filtering based on a custom input. Unlike PivotFilters, slicers have an [Excel UI component](https://support.office.com/article/Use-slicers-to-filter-data-249f966b-a9d5-4b0f-b31a-12651785d29d). With the `Slicer` class, you create this UI component, manage filtering, and can control its visual appearance. 
+
+See [Filter with PivotFilters](#filter-with-pivotfilters) and [Filter with slicers](#filter-with-slicers) to learn more about filtering data with Excel JavaScript APIs.
```

Comment:
I'm not sure if you need this line, since both those sections are contained within this section.